### PR TITLE
Release 1.23.0

### PR DESCRIPTION
**Pull request by khorne3**
## Release Checklist

  - [x] All feature documentation reviewed and merged with `navigable` set to `false`
  - [x] Changelog completed
  - [x] Documentation owner review
  - [x] Release manager review
  - [x] Create branch for prior version (`v1.22`)
  - [x] Update `manifest.json` to show newly added feature documentation (set `navigable` to `true`)

### CODE REVIEW

**Comment by khorne3** [comment]
Outstanding: docs for generic apps and a guide for Terraform (the latter isn't a blocker)

File: changelog/1.23.0.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+- Doctor: Doctor assesses a Kubernetes cluster to determine its readiness for
+  Coder installation; if there are issues, Doctor provides information on why.
+- Terraform: Coder has released the first of a series of
+  [Terraform scripts](https://github.com/cdr/enterprise-terraform) to facilitate
+  one-click Coder deployment.
+
+- web: added support for generic applications; users can provide images with
+  custom applications, instead of only using applications that have been built
+  into the container image `PATH`.
```
Comment (scsmithr) [inline changelog/1.23.0.md:20]: Would we be able to link to the docs here?

Comment (khorne3) [reply changelog/1.23.0.md:20]: We can as soon as they're ready

File: changelog/1.23.0.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+- web: added ability to specify the Kubernetes service account name when editing
+  a workspace provider.
+- cli: added `run coder update` command; users can now update the Coder CLI
+  directly.
+- api: added ability to use the Coder API to create workspaces.
+- infra: updated logging so that Coder will send an error to `stdout` if unable
+  to write to the in-product audit log.
+- infra: add support for containerd for cached CVMs.
+- infra: updated Coder to always use the latest stable version of JetBrains
+  Projector.
```
Comment (bpmct) [inline changelog/1.23.0.md:47]: @jawnsy pointed out we may not "always" version bump before a release. @Emyrk can you confirm?

Comment (khorne3) [reply changelog/1.23.0.md:47]: We can omit always?

Comment (deansheather) [reply changelog/1.23.0.md:47]: Let's omit always.

File: changelog/1.23.0.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  CVMs.
+- infra: fixed issue with cached CVMs not having internet connectivity when
+  network policies are enabled on GKE.
+- infra: fixed issue with cached CVMs not preserving environment variables
+  passed from the image.
+- infra: consolidated requests sent when to get dev URL status, lessening server
+  load.
+- infra: fixed goroutine/memory leaks.
+- infra: fixed issue with inability to delete workspace provider if its cluster
+- has been deleted.
```
Comment (deansheather) [inline changelog/1.23.0.md:74]:
```suggestion
  has been deleted.
```

File: changelog/1.23.0.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  CVMs.
+- infra: fixed issue with SSH connections immediately closing when using cached
+  CVMs.
+- infra: fixed issue with cached CVMs not having internet connectivity when
+  network policies are enabled on GKE.
+- infra: fixed issue with cached CVMs not preserving environment variables
+  passed from the image.
+- infra: consolidated requests sent when to get dev URL status, lessening server
+  load.
+- infra: fixed goroutine/memory leaks.
```
Comment (deansheather) [inline changelog/1.23.0.md:72]:
```suggestion
```

No new leaks fixed since 1.22.1.

**Review by deansheather — APPROVED** [review]

File: changelog/1.23.0.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+### Features ✨
+
+- Doctor: [Doctor](https://github.com/cdr/coder-doctor) assesses a Kubernetes
+  cluster to determine its readiness for Coder installation; if there are
+  issues, Doctor provides information on why.
+- Terraform: Coder has released the first of a series of
+  [Terraform scripts](https://github.com/cdr/enterprise-terraform) to facilitate
+  one-click Coder deployment.
+
```
Comment (scsmithr) [inline changelog/1.23.0.md:18]:
```suggestion
```

I think this newline is causing the markdown renderer to insert extra paragraph elements:
image

Comment (scsmithr) [inline changelog/1.23.0.md:18]:
```suggestion
```

I think this newline is causing the markdown renderer to insert extra paragraph elements:
image

File: changelog/1.23.0.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+### Features ✨
+
+- Doctor: [Doctor](https://github.com/cdr/coder-doctor) assesses a Kubernetes
+  cluster to determine its readiness for Coder installation; if there are
+  issues, Doctor provides information on why.
+- Terraform: Coder has released the first of a series of
+  [Terraform scripts](https://github.com/cdr/enterprise-terraform) to facilitate
+  one-click Coder deployment.
+- web: added support for [generic applications](../workspaces/applications.md);
```
Comment (scsmithr) [inline changelog/1.23.0.md:18]:
```suggestion
- web: added support for [workspace applications](../workspaces/applications.md);
```

File: workspaces/applications.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+![Application Launcher](../assets/workspaces/applications.png)
+
+## Enabling custom apps
+
+If you'd like to use custom apps in Coder, you can enable this feature in the
+UI:
+
+1. In the top-right, click on your avatar and select **Feature Preview**
+1. Click **Generic applications** and select **Enable**.
```
Comment (scsmithr) [inline workspaces/applications.md:19]:
```suggestion
1. Click **Workspace applications** and select **Enable**.
```

**Pull request merged**